frontend/src/components/pages/ChangePassword/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import Presentation from "./Presentation";
 
-export default () => {
+export default function () {
```

Comment:
The style guide says that functions should have names. Eg:

```

function Listing({ hello }) {
    return <div>{hello}</div>;
}
```

This is so that the ReactDevTools can be useful. If a component has no name then it just gets called AnonomousComponent